docs/dev/adding-new-platform.md

```diff
@@ -0,0 +1,158 @@
+8. **Publish Images** Ensure RH CoreOS images on the platform are being published to a location CI can test
+9. **Enable** Merge the platform PR to the installer with a green passing CI job
+
+At this point the platform is said to be an `unsupported IPI` (installer provided infrastructure) install - work can begin enabling in other repositories. Once documentation is completed for step 2 as part of the official platform bring up, the platform can said to be `supported UPI` (user provided infrastructure) for the set of options in **Arch**
+
+### Enable day 2
+
+Once the platform can be booted and tested, day 2 automation can be enabled in individual operators. The items below are unordered because they can be completed in parallel.
+
+* Ensure the cloud provider is enabled in Kubernetes for your platform
+* Enable dynamic compute (MachineSets) by adding a cloud actuator for that platform
+* Enable the cloud credential operator for the platform
+* Enable cluster-ingress-operator to provision the wildcard domain names
+* Enable cluster-image-registry-operator to provision a storage bucket (if your platform supports object storage)
+* Enable cluster-storage-operator to set the correct default storage class
+
+Other important day 2 changes:
+
+* Replace the installer terraform destroy with one that doesn't rely on terraform state
+* Ensure the installer IPI support is consistent with other platform features (private config, etc)
+* Enable a CI job that verifies the e2e suite for the given platform runs
+
+At this point the platform is said to be a `supported IPI with Dynamic Compute` if the platform supports
+MachineSets, or `supported IPI without Dynamic Compute` if it does not.
+
+
+Defining the supported architecture for a platform
+--------------------------------------------------
+
+An OpenShift cluster programs the infrastructure it runs on to provide operational simplicity. For every platform, the minimum requirements are:
+
+1. The control plane nodes:
+   1. Run RH CoreOS, allowing in-place updates
+   2. Are fronted by a load balancer that allows raw TCP connections to port 6443 and exposes port 443
+   3. Have low latency interconnections connections (<5ms RTT) and persistent disks that survive reboot and are provisoned for at least 300 IOPS
+   4. Have cloud or infrastructure firewall rules that at minimum allow the standard ports to be opened (see AWS provider)
+   5. Do *not* have automatic cloud provider permissions to perform infrastructure API calls
+   6. Have a domain name pointing to the load balancer IP(s) that is `api.<BASE_DOMAIN>`
+   7. Has an internal DNS CNAME pointing to each master called `etcd-N.<BASE_DOMAIN>` that 
+   8. Has an optional internal load balancer that TCP load balances all master nodes, with a DNS name `internal-api.<BASE_DOMAIN>` pointing to the load balancer.
```

Comment:
Is the DNS name optional too (or just the load balancer)? Would the external DNS need the internal-api name registered for use by the cluster without internal DNS?

Reply:
DNS isn't optional for cert signing, but I guess you could technically sign your IP.